ww4ff/data/transform/augment.py

```diff
@@ -0,0 +1,292 @@
+from dataclasses import dataclass
+from typing import Sequence
+import math
+import random
+
+from torchaudio.transforms import MelSpectrogram, ComputeDeltas
+import numpy as np
+import librosa
+import torch
+import torch.nn as nn
+
+from ww4ff.data.dataset import EmplacableExample
+
+
+__all__ = ['AugmentationParameter',
+           'AugmentModule',
+           'TimeshiftTransform',
+           'TimestretchTransform',
+           'NoiseTransform',
+           'StandardAudioTransform',
+           'SpecAugmentTransform']
+
+
+@dataclass
+class AugmentationParameter(object):
+    domain: Sequence[float]
+    name: str
+    current_value_idx: int = None
```

Comment:
I guess we don't really need domain/current_value_idx stuff because this was meant to be used for squawk where the main goal is to find the best parameter.

Reply:
I'll keep it around since we'll do random search over the domain later.